tools/relay.js

```diff
@@ -0,0 +1,31 @@
+const { Relay } = require('bedrock-protocol')
```

Comment:
Remove file

Reply:
ah, just thought it would be useful to have in the environment